react/daily-art-prompt/src/store/rootReducer.test.js

```diff
@@ -447,32 +447,32 @@ describe('indexReducer', () => {
     describe('when action.type equals UPDATE_NEXT_INDEX', () => {
         describe('when there is a next prompt', () => {
             it('should return newIndex', () => {
-                const state = [
-                    {
-                        index: 0
-                    }
-                ]
+                const state =
+                {
+                    index: 0
+                }
                 const action = {
                     type: TYPES.UPDATE_NEXT_INDEX,
                     payload: {
-                        index: state[0].index
+                        index: state.index,
+                        promptslength: 2
                     }
                 }
-                expect(indexReducer(state, action)).toEqual(state[0].index + 1)
+                expect(indexReducer(state, action)).toEqual(state.index + 1)
             })
         })
 
         describe('when there is no next prompt', () => {
             it('should return state unchanged', () => {
-                const state = [
-                    {
-                        index: 2
-                    }
-                ]
+                const state =
+                {
+                    index: 2
+                }
                 const action = {
                     type: TYPES.UPDATE_NEXT_INDEX,
                     payload: {
-                        index: 2
+                        index: 2,
+                        promptslength: 1
```

Comment:
maybe we should test when the promptsLength is the same as the index. Its more clear that way

Reply:
done